src/WebJobs.Script.WebHost/Standby/StandbyManager.cs

```diff
@@ -34,17 +41,18 @@ public class StandbyManager : IStandbyManager
         private static IChangeToken _standbyChangeToken = new CancellationChangeToken(_standbyCancellationTokenSource.Token);
         private static SemaphoreSlim _semaphore = new SemaphoreSlim(1, 1);
 
-        public StandbyManager(IScriptHostManager scriptHostManager, IConfiguration configuration, IScriptWebHostEnvironment webHostEnvironment,
+        public StandbyManager(IScriptHostManager scriptHostManager, ILanguageWorkerChannelManager languageWorkerChannelManager, IServiceProvider rootServiceProvider, IConfiguration configuration, IScriptWebHostEnvironment webHostEnvironment,
             IEnvironment environment, IOptionsMonitor<ScriptApplicationHostOptions> options, ILogger<StandbyManager> logger)
         {
             _scriptHostManager = scriptHostManager ?? throw new ArgumentNullException(nameof(scriptHostManager));
             _options = options ?? throw new ArgumentNullException(nameof(options));
+            _eventManager = (IScriptEventManager)rootServiceProvider.GetService(typeof(IScriptEventManager));
```

Comment:
This should not have a dependency on the `IServiceProvider`, the dependency on `IScriptEventManager` should be explicit here.